src/componentIcons/icon-loading-spinner.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg aria-hidden="true" viewBox="0 0 40 40" version="1.1">
```

Comment:
I think the `aria-hidden` will be handled in ui-components in the icon component, so we shouldn't have it hard-coded here.